en/flight_controller/durandal.md

```diff
@@ -0,0 +1,365 @@
+# Holybro Durandal
+
+*Durandal*<sup>&reg;</sup> is the latest update to the successful family of Holybro flight controllers. 
+It was designed and developed by Holybro.
+
+![Durandal](../../assets/flight_controller/durandal/durandal_hero.jpg)
+
+At high level, some of the key features are:
+- Integrated temperature control for sensors.
+- Powerful STM32H7 microcontroller running at 480MHz.
+  2 MB of Flash memory and 1 MB of RAM.
+- New sensors with higher temperature stability.
+- Efficient vibration absorption system.
+- Dual high-performance, low-noise IMUs on board are designed for demanding stabilization applications.
+
+A summary of the key features, [assembly](../assembly/quick_start_durandal.md), and [purchase](#purchase) links can be found below.
+
+
+## Quick Summary
+
+- Main FMU Processor: STM32H743
+  - 32 Bit Arm ® Cortex® -M7, 480MHz, 2MB memory, 1MB RAM
+- IO Processor: STM32F100
+  - 32 Bit Arm ® Cortex® -M3, 24MHz, 8KB SRAM
+- On-board sensors
+  - Accel/Gyro: ICM-20689
+  - Accel/Gyro: BMI088
+  - Mag: IST8310
+  - Barometer: MS5611
+- GPS: ublox Neo-M8N GPS/GLONASS receiver; integrated magnetometer IST8310
+
+- Interfaces:
+  - 8-13 PWM servo outputs (8 from IO, 5 from FMU)
+  - 6 dedicated PWM/Capture inputs on FMU 
+  - Dedicated R/C input for Spektrum / DSM
+  - Dedicated R/C input for CPPM and S.Bus
+  - Dedicated S.Bus servo output and analog / PWM RSSI input
+  - 5 general purpose serial ports
+    - 3 with full flow control
+    - 1 with separate 1.5A current limit
+  - 3 I2C ports
+  - 4 SPI buses
+    - 1 internal high speed SPI sensor bus with 4 chip  selects and 6 DRDYs
+    - 1 internal low noise SPI bus dedicated for XXX
+    - Barometer with 2 chip selects, no DRDYs
+    - 1 internal SPI bus dedicated for FRAM
+    - Supports temperature control located on sensor module
+    - 1 external SPI buses
+  - Up to 2 CANBuses for dual CAN
+    - Each CANBus has individual silent controls or ESC RX-MUX control
+  - Analog inputs for voltage / current of 2 batteries
+  - 2 additional analog inputs
+
+- Electrical Data:
+  - Power module output: 4.9~5.5V
+  - Max input voltage: 6V
+  - Max current sensing: 120A
+  - USB Power Input: 4.75~5.25V
+  - Servo Rail Input: 0~36V
+
+- Mechanical Data:
+  - Dimensions: 80x45x20.5mm
+  - Weight: 68.8g
+  
+- Other Characteristics:
+  - Operating temperature: ~40~85C
+  - Storage temperature: -40~85C
+  - CE
+  - FCC
+  - RoHS compliant (lead-free)
+
+For more information see: [Durandal Technical Data Sheet](http://www.holybro.com/manual/Durandal_technical_data_sheet.pdf).
+
+## Purchase {#purchase}
+
+Order from [Holybro](https://shop.holybro.com/durandalbeta_p1189.html).
+
+
+## Connectors
+
+The connectors are shown in the [pinouts section](#pinouts) below.
+
+
+## Dimensions
+
+All dimensions are in millimeters.
+
+![Durandal Dimensions](../../assets/flight_controller/durandal/durandal_dimensions.jpg)
+
+<!-- 
```

Comment:
Below is voltage rating stuff from Pixhawk 4. Is there/should there be equivalent for Durandal? Ie can it be redundant? What do I need to mention here? Commented out for now.

Reply:
They are likely the same, but does not have to be (it has 2 power inputs as well, but the limits might be different). @davids5 can you have a look?